outputs.tf

```diff
@@ -1,5 +1,24 @@
+output "result" {
+  description = "String result of Lambda execution"
+  value       = module.lambda_invocation_result.stdout
+}
+
+output "invocation_stdout" {
```

Comment:
Are these just in for debugging purposes? If so maybe nix? If they are worth keeping, that's fine. Seems weird though. Maybe a TODO explaining?

Reply:
They are attempting to control the dependency flow with some of the modules I'm using. TF 0.13.0 will make this a lot better. I'll put in a comment.